### DISCUSSION TITLE
2023년 08월 29일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#76] [General]
# 2023년 08월 29일

LOG 오늘 하려고 목표했던 일은 잘 끝냈는데, 시간 계산을 잘못한 탓인지 6시에 가야하는 약속에 1시간이나 늦어버렸다... 너무너무 미안하기도 하고 하필 비오는 날이라 버스랑 지하철도 밀려서 마음이 너무 안 좋았다. 그에 더해서 (오늘 시간이 많았…

https://vero.wiki/2023-08-29/

**Comment by ukkodeveloper** [comment]
베로 화이팅!!! 항상 응원해~

**Reply by Cyma-s** [reply]
이런 글에 댓글이라니 갬동이야 🥺

**Comment by somsom13** [comment]
괜찮아 귀여운 아기

**Reply by Cyma-s** [reply]
역시 내가 귀여운 탓인가... 🥹
